phpstan.neon.dist

```diff
@@ -1,9 +1,14 @@
 parameters:
-    level: 5
+    level: 6
     bootstrapFiles:
         - tests/bootstrap_phpstan.php
     paths:
-        - src
+        #- src
```

Comment:
Very nice, but you disabled src though :)

Reply:
Indeed, but it is coming back very shortly (I didn't want to do a massive PR with everything in one go).